public/server.py

```diff
@@ -1,17 +1,15 @@
 #!/usr/bin/env python
-import http.server
-import socketserver
-class MyHttpRequestHandler(http.server.SimpleHTTPRequestHandler):
+import SimpleHTTPServer, SocketServer
```

Comment:
Does this need to be run using python2? Why is that?

Using python3 I had to change it back to:
`import http.server, socketserver`

Reply:
Good q - the EC2 we use has python 2 by default (as does my Mac :) )